core/backend/src/Texture.ts

```diff
@@ -23,27 +33,30 @@ export class Texture extends DefinitionElement {
   public height: number;
   public flags: TextureFlags;
   public description?: string;
+
   /** @internal */
-  constructor(props: TextureProps & { data: Uint8Array | string }, iModel: IModelDb) {
+  constructor(props: TextureCreateProps, iModel: IModelDb) {
     super(props, iModel);
     this.format = props.format;
-    this.data = (typeof props.data == "string") ? Buffer.from(props.data, "base64") : props.data;
+    this.data = typeof props.data === "string" ? Base64EncodedString.toUint8Array(props.data) : props.data;

```

Comment:
wow, how did that get through our linter?

Reply:
Maybe it recognizes that comparing `typeof` to `"string"` is always unambiguous, I don't know.